sdk/core/System.ClientModel/src/Internal/BitVector640.cs

```diff
@@ -16,7 +16,7 @@ namespace System.ClientModel.Internal;
 /// </summary>
 internal struct BitVector640
 {
-    // Keeping ulongs as fields puts them on the stack.
+    // Ten unsigned long brackets to keep 640 bits.
```

Comment:
I find the name "bracket" confusing. It's new terminology that is not necessary IMO. You call them "unsigned long fileds" above, and I would stick to one term to refer to them.

Reply:
Please don't introduce new concept names.  Let's use words like fields that are part of the language.

Reply:
It would also be good to retain the comment that explains why we want to use fields instead of objects on the heap.

Reply:
> retain the comment that explains why we want to use fields instead of objects on the heap.

This comment is invalid. `BitVector640` is not a `ref struct`.

Reply:
> This comment is invalid. BitVector640 is not a ref struct.

I'm happy to update my mental model on this, but I had thought fields of a struct went on the stack when it was created as a local variable.  Can you point me to documentation so I can learn the nuances I'm missing here?

Reply:
> I would stick to one term to refer to them.

Done

Reply:
> > This comment is invalid. BitVector640 is not a ref struct.
> 
> I'm happy to update my mental model on this, but I had thought fields of a struct went on the stack when it was created as a local variable. Can you point me to documentation so I can learn the nuances I'm missing here?

Unless it is a `ref struct`, we 

> > This comment is invalid. BitVector640 is not a ref struct.
> 
> I'm happy to update my mental model on this, but I had thought fields of a struct went on the stack when it was created as a local variable. Can you point me to documentation so I can learn the nuances I'm missing here?

Unless it is a `ref struct`, you can't control if the struct is in the stack or in the heap. In fact, the only usage we have is a field in the [ResponseStatusClassifier](https://github.com/Azure/azure-sdk-for-net/blob/cc45f7e43aac5737f05d802a1fb2c4fa40bd2098/sdk/core/System.ClientModel/src/Options/ResponseStatusClassifier.cs#L11)